catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_chm.jsonnet

```diff
@@ -0,0 +1,192 @@
+local id = 'projects/neon-prod-earthengine/assets/chm';
+local subdir = 'neon-prod-earthengine';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+local units = import 'units.libsonnet';
+
+{
+  // TODO(schwehr): Remove when the dataset is ready.
+  'gee:skip_indexing': true,
+
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
```

Comment:
We included these stac_version, etc. because we saw these in other examples, but not sure if they are required, or what exactly they mean.

Reply:
We usually recommend choosing a version anyway (v1, if nothing else sounds better) because data sometimes get reprocessed, and it would be easier for users to navigate different versions if the version number is explicit. But this is not a hard requirement.